stylesheets/_conversation.scss

```diff
@@ -720,7 +720,9 @@ li.entry .error-icon-container {
       height: 100%;
       width: 100%;
     }
-
+    .icon:hover {
+      background-color: #616161;
+    }
```

Comment:
Looks good. Tested it, works out, but there's one small flaw: It's not working with dark theme. Could you add that as well?